internal/controllers/machine/machine_controller_noderef.go

```diff
@@ -116,6 +117,13 @@ func (r *Reconciler) reconcileNode(ctx context.Context, cluster *clusterv1.Clust
 		}
 	}
 
+	if reconcileManagedLabels(machine, node) {
+		options := []client.PatchOption{
+			client.FieldOwner("MachineController"),
```

Comment:
```suggestion
			client.FieldOwner("MachineController"),
```
nit

We started with SSA by using "capi-topology" as field owner for the Cluster topology controller.

Wonder if we should follow that "pattern". Maybe "capi-machine" ?